create_cluster/tasks/main.yml

```diff
@@ -1,43 +1,38 @@
 ---
 # tasks file for create_cluster
-
+  
```

Comment:
IMO unnecessary whitespace changes should be avoided, especially when working with a format with meaningful whitespace. 
It usually makes the changeset harder to understand and has the potential to introduce unintended changes in behaviour when whitespace is meaningful, as it is in yaml.